department-of-veterans-affairs/va.gov-cms
# VACMS-4049: Policy-Tooltips-Accessible

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/4049

## Testing done


## Screenshots


## QA steps

- Navigate to preview environment
- Navigate here: `/admin/structure/types/manage/vamc_system_policies_page/form-display`
---
- Confirm the following **3** new `Tooltip` field groups:
- [ ] National top of page content
- [ ] National general visitation policy
- [ ] National bottom of page content
---
- Select the gear icon for each of the field groups from above and confirm the following fields:
![image](https://user-images.githubusercontent.com/56809719/107565747-02c8d900-6bb2-11eb-8876-3d95c3c69ecf.png)
---
- On a separate tab navigate here: `/node/add/vamc_system_policies_page`
- [ ] Confirm the tooltip appears and that the text within the tooltip matches the `Tooltip Description` from the image above.
- [ ] Complete form, save and confirm the Tooltips appear and function as expected.
- [ ]  using the tab button on your keyboard validate that the tooltip activates with tab and reflects the order of the content.
- [ ] Inspect element to confirm accessibility issues are resolved.
---
- Navigate back to the manage form display tab
- Select the gear setting next for `National top of page content`
- Change the `Show Label` field to `Yes`
- Copy the `Tooltip Description` over to the `Description` textarea. 
- Save
---
- Navigate back here: `/node/add/vamc_system_policies_page`
- [ ] Confirm the tooltip does NOT appear.
- [ ] Confirm the description now appears under the field label.
- [ ] Inspect element to confirm accessibility issues are resolved.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
